frontend/packages/dev-console/src/components/health-checks/health-checks-types.ts

```diff
@@ -0,0 +1,39 @@
+import { NameValuePair } from '@console/shared';
+
+export enum HealthChecksProbeType {
+  ReadinessProbe = 'readinessProbe',
+  LivenessProbe = 'livenessProbe',
+  StartupProbe = 'startupProbe',
+}
+
+export enum ProbeType {
+  HTTPGET = 'httpGet',
+  ContainerCommand = 'command',
+  TCPSocket = 'tcpSocket',
+}
+
+export interface HealthCheckProbeDataType {
+  failureThreshold: number;
+  probeType?: string;
+  httpGet?: {
+    scheme: string;
+    path: string;
+
+    port: number;
+    httpHeaders: NameValuePair[];
+  };
+  tcpSocket?: {
+    port: number;
+  };
+  command?: string[];
+  initialDelaySeconds: number;
+  periodSeconds: number;
+  timeoutSeconds: number;
+  successThreshold: number;
+}
+
+export interface HealthCheckProbeType {
```

Comment:
You have an enum with almost same name `HealthChecksProbeType`. Its very confusing.
```suggestion
export interface HealthCheckProbe {
```